packages/7z-nsis.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,43 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = '7z'
+    $category = 'Productivity Tools'
+
+    $url = 'https://github.com/myfreeer/7z-build-nsis/releases/download/23.01/7z2301-x86.exe'
+    $checksum = '7b1d50073e6d3631267f2bbb986fb1faffddc5fc72d6bc23e10b5920a6f365b4'
+    $url64 = 'https://github.com/myfreeer/7z-build-nsis/releases/download/23.01/7z2301-x64.exe'
+    $checksum64 = 'b7f1d8360d988808447f9af3989db7665dfec72bac83c8b6467cc35f8fe718ff'
+
+    $packageArgs = @{
+    packageName   = ${Env:ChocolateyPackageName}
+    fileType      = 'EXE'
+    url           = $url
+    url64bit      = $url64
+    checksumType  = 'sha256'
+    checksum      = $checksum
+    checksum64    = $checksum64
+    silentArgs    = '/S'
+    }
+    Install-ChocolateyPackage @packageArgs
+
+    $toolDir = Join-Path ${Env:ProgramFiles} '7-Zip' -Resolve
+    $7zExecutablePath = Join-Path $toolDir "$toolName.exe" -Resolve
+    Install-BinFile -Name $toolName -Path $7zExecutablePath
+
+    # Make shortcut point to 7z File Manager so that it's more useful of a shortcut.
+    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+    $executablePath = Join-Path $toolDir ($toolName + "FM.exe") -Resolve
+    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath
```

Comment:
what about using `VM-Install-Shortcut` for consistency?

Reply:
Yup! For some reason my brain thought it couldn't do a shortcut to `${toolName}FM.exe` using that...but I am definitely wrong about that. 😅